src/components/user/repo/account.repo.js

```diff
@@ -0,0 +1,32 @@
+const accountModel = require('../../../services/account/model');
```

Comment:
I really like that you started implementing repos, and long term this is as it should be. Whoever, I think it is even more important to restrict direct access to the actual mongoose model to a single entity. Until we can get rid of the account modelservice, it should remain that entity.
In the meantime please access the modelservice instead of the model itself, and maybe add a comment that this should be changed in the future.

Reply:
done